reference/OpenAPI.json

```diff
@@ -13305,22 +13305,22 @@
           "discount": {
             "anyOf": [
               {
-                "$ref": "#/components/schemas/6_res_validate_voucher_discount_amount"
+                "$ref": "#/components/schemas/1_obj_voucher_object_discount_amount"
               },
               {
-                "$ref": "#/components/schemas/6_res_validate_voucher_discount_percent"
+                "$ref": "#/components/schemas/1_obj_voucher_object_discount_percentage"
               },
               {
-                "$ref": "#/components/schemas/6_res_validate_voucher_discount_fixed"
+                "$ref": "#/components/schemas/1_obj_voucher_object_discount_fixed"
               },
               {
-                "$ref": "#/components/schemas/6_res_validate_voucher_discount_unit"
+                "$ref": "#/components/schemas/1_obj_voucher_object_discount_unit_one"
               },
               {
-                "$ref": "#/components/schemas/6_res_validate_voucher_discount_unit_multiple"
+                "$ref": "#/components/schemas/1_obj_voucher_object_discount_unit_multiple"
               },
               {
-                "$ref": "#/components/schemas/6_res_validate_voucher_discount_shipping"
+                "$ref": "#/components/schemas/1_obj_voucher_object_discount_shipping"
               }
```

Comment:
The responses for a validation is not the same as the discount definition. These objects are different in the context they are in. The ones with 1_ prefix define the discount, the ones with 6_res provide the discount behavior. To see the difference, send a response do the [Get Voucher](https://docs.voucherify.io/reference/get-voucher) endpoint & send the same voucher in a [voucher validation](https://docs.voucherify.io/reference/validate-voucher) request. 

### Get Voucher

```json
 "discount": {
        "type": "PERCENT",
        "percent_off": 20,
        "percent_off_formula": "ORDER_AMOUNT * 0.9",
        "effect": "APPLY_TO_ORDER"
    }
```

### Validation Request

```json
 "discount": {
        "type": "PERCENT",
        "effect": "APPLY_TO_ORDER",
        "percent_off": 100,
        "is_dynamic": true
    }
```
Having said that, I will update the 6_res objects because I can see that there is now a `is_dynamic` boolean property.